test/lib/graph/helpers/ui5Framework.js

```diff
@@ -177,7 +178,7 @@ test.serial("enrichProjectGraph: without framework configuration", async (t) =>
 
 test.serial("enrichProjectGraph SNAPSHOT", async (t) => {
 	const {sinon, ui5Framework, utils, Sapui5MavenSnapshotResolverInstallStub} = t.context;
-	process.env.UI5_MAVEN_SNAPSHOT_ENDPOINT = "__url__";
+	process.env.UI5_MAVEN_SNAPSHOT_ENDPOINT_URL = "__url__";
```

Comment:
Is this even required within this test? I wonder why it didn't fail before, as the variable has been renamed already.

Reply:
Done